.appveyor.yml

```diff
@@ -85,8 +85,8 @@ build_script:
   - cd C:\projects\mlpack && mkdir build && cd build
   - >
     cmake -G "%VSVER%"
-    -DBLAS_LIBRARY:FILEPATH=%BLAS_LIBRARY%
-    -DLAPACK_LIBRARY:FILEPATH=%BLAS_LIBRARY%
+    -DBLAS_LIBRARIES:FILEPATH=%BLAS_LIBRARY%
+    -DLAPACK_LIBRARIES:FILEPATH=%BLAS_LIBRARY%
```

Comment:
I'm a bit confused on how `BLAS_LIBRARIES` and `LAPACK_LIBRARIES` get used eventually be a part of the variable `ARMADILLO_LIBRARIES`, which is what is actually used in the various `add_library()` commands to link against either Armadillo or all of its dependencies (depending on the status of `ARMA_USE_WRAPPER`).

Reply:
This one was my own misunderstanding---it makes way more sense now. :+1: